WalletWasabi.Backend/wwwroot/index.html

```diff
@@ -50,22 +50,22 @@
                 <div class="collapse navbar-collapse pt-2 pt-sm-0" id="navbarColor02">
                     <ul class="navbar-nav ml-auto">
                         <li class="nav-item">
-                            <a class="nav-link" href="#faq">faq</a>
+                            <a class="nav-link" href="#faq" title="FAQs">faq</a>
```

Comment:
I believe it should be:

```suggestion
                            <a class="nav-link" href="#faq" title="FAQ">faq</a>
```

Not sure. Feel free to ignore.

Reply:
There are multiple frequently asked questions, so I think we should use the plural.

Not only for the `title` attribute, but also for the label itself

Reply:
![image](https://user-images.githubusercontent.com/58662979/93887089-87d85200-fce6-11ea-86c0-4d4af9371044.png)

I don't know I understand that FAQ is reserved for a list of questions. FAQs is reserved for multiple lists of questions. Not sure that Wasabi FAQ actually contains.